### PR TITLE
Add CITATION.cff file

### PR DESCRIPTION
**Pull request by mikemhenry**
Resolves: #99

### CODE REVIEW

**Comment by mikemhenry** [comment]
The author list order is arbitrary, except for putting our fearless leader first :smile:

**Comment by richardjgowers** [comment]
@IAlibay @dwhswenson Can you double check yours?

**Review by IAlibay — APPROVED** [review]
lgtm

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+authors:
+- family-names: "Gowers"
+  given-names: "Richard J."
+  orcid: "https://orcid.org/0000-0002-3241-1846"
+- family-names: "Alibay"
+  given-names: "Irfan"
+  orcid: "https://orcid.org/0000-0001-5787-9130"
+- family-names: "Swenson"
+  given-names: "David W.H."
+  orcid: "https://orcid.org/0000-0001-9922-7923"
```
[the remark this answers is not in the document]
Comment (mikemhenry) [reply CITATION.cff:12]: Can do!

**Review by richardjgowers — APPROVED** [review]

**Pull request merged**